models/projection/powder/mantid_projection_calculator.py

```diff
@@ -1,22 +1,11 @@
-from math import sqrt,cos
-
-from mantid.simpleapi import SofQWNormalisedPolygon,mtd
-
+from mantid.simpleapi import ConvertToMD
 from models.projection.powder.projection_calculator import ProjectionCalculator
 
 
 class MantidProjectionCalculator(ProjectionCalculator):
     def calculate_projection(self, input_workspace, output_workspace, qbinning, axis1, axis2):
-        if axis1 == 'Energy' and axis2 == '|Q|':
-            #TODO is EMode always direct ?
-            SofQWNormalisedPolygon(InputWorkspace=input_workspace, OutputWorkspace=output_workspace,
-                                                                QAxisBinning=qbinning,EMode='Direct')
+        if axis1 == '|Q|' and axis2 == 'Energy':
+            ConvertToMD(InputWorkspace=input_workspace, OutputWorkspace=input_workspace+'_MD',QDimensions='|Q|')
```

Comment:
I think it would be good to pull out the suffix string, `_MD`, as a module-level variable so that it is clearer where it would be changed, e.g.
```python
MD_SUFFIX = '_MD'
```
above the class definition similar to this type of thing: https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/backends/backend_qt5.py#L73